sources/procfs.go

```diff
@@ -207,10 +207,16 @@ func (s *lustreSource) generateOSTMetricTemplates() error {
 			{"lock_timeouts", "lock_timeout_total", "Number of lock timeouts", s.counterMetric},
 			{"contended_locks", "lock_contended_total", "Number of contended locks", s.counterMetric},
 			{"contention_seconds", "lock_contention_seconds_total", "Time in seconds during which locks were contended", s.counterMetric},
+			{"pool/cancel_rate", "lock_cancel_rate", "Lock cancel rate", s.gaugeMetric},
 			{"pool/granted", "locks_granted_total", "Number of granted locks", s.counterMetric},
+			{"pool/grant_plan", "lock_grant_plan", "Number of planned lock grants for time period", s.gaugeMetric},
```

Comment:
Do we know what the time period is? We should clarify that...otherwise we may not understand the metric.